portfolio/src/main/webapp/script.js

```diff
@@ -36,8 +36,22 @@ function myFunction() {
   document.getElementById("myBar").style.width = scrolled + "%";
 }
 
-async function getMessage() {
-    const response = await fetch('/data');
-    const message = await response.text();
-    document.getElementById('message-container').innerHTML = message;
+function getMessage() {
+    fetch('/data').then(response => response.json()).then((someMessages) => {
```

Comment:
I'm curious: why did you switch from the `async` / `await` form?

Reply:
Um, I am guessing it was a matter of the tutorial I was following. I thought this was a more quick way of doing all in one step, so it seemed more simple to understand and to write.